packages/ui/src/components/sign-up/sign-up.tsx

```diff
@@ -74,7 +74,7 @@ export function SignUpComponent() {
                 asChild
               >
                 <Button
-                  icon={<Icon.CaretRight />}
+                  icon={<Icon.CaretRight className='rtl:rotate-180' />}
```

Comment:
💭 wonder if internally the CaretRight should handle this rotation. Not all icons should rotate so it'd only be directional icons.

Reply:
Felt like the right approach https://github.com/clerk/javascript/pull/3505/commits/532d8e3d2338aa374d4a0b14168417d199555939